new-cdc/eventuate-local-java-cdc-connector-mysql-binlog/build.gradle

```diff
@@ -10,6 +10,15 @@ dependencies {
     testCompile "org.springframework.boot:spring-boot-starter-test:$springBootVersion"
 }
 
+test {
```

Comment:
This is ugly and suggests a lack of modularity: binlog code should be in a different module than say Kafka code.